plugins/scaffolder/src/components/TaskPage/TaskPage.tsx

```diff
@@ -351,6 +360,15 @@ export const TaskPage = ({ loadingText }: TaskPageProps) => {
                   >
                     Start Over
                   </Button>
+                  <Button
+                    className={classes.button}
+                    onClick={handleAbort}
+                    disabled={completed}
+                    variant="contained"
```

Comment:
I'm feeling this is perhaps a bit too visible, perhaps make it outlined instead?